qualtran/bloqs/mod_arithmetic/mod_addition.py

```diff
@@ -300,6 +352,12 @@ class CtrlScaleModAdd(Bloq):
         ctrl: The control bit
         x: The 'source' quantum register containing the integer to be scaled and added to `y`.
         y: The 'destination' quantum register to which the addition will apply.
+
+    References:
+        [How to factor 2048 bit RSA integers in 8 hours using 20 million noisy qubits](https://arxiv.org/abs/1905.09749).
+        Construction based on description in section 2.2 paragraph 4. We add n And/And† bloqs
+        because the bloq is controlled, but the construction also involves modular addition
+        controlled on the qubits comprising register x.
```

Comment:
I don't understand what this is trying to say. Please try to disambiguate between the construction in Qualtran and the construction in the reference

Reply:
I tried to be more clear. Basically, the reference implementation uses CModAddK, but the circuit that it points to is just ModAdd (not ModAddK). This ModAdd is less efficient than the circuit later introduced in the Litinski paper so we choose to use that since it is more efficient and already implemented in Qualtran.

Reply:
can you just write that instead?

This is my secret trick for good technical writing: write all your docstrings as if you're explaining something to an actual human reading your code; because spoiler alert: that's what you're doing!

Reply:
Simplified further and used above explanation.